src/main/java/com/cloudbees/jenkins/GitHubWebHookCrumbExclusion.java

```diff
@@ -9,18 +9,25 @@
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
 
+import static org.apache.commons.lang3.StringUtils.isEmpty;
+
 @Extension
 public class GitHubWebHookCrumbExclusion extends CrumbExclusion {
 
     @Override
     public boolean process(HttpServletRequest req, HttpServletResponse resp, FilterChain chain)
             throws IOException, ServletException {
         String pathInfo = req.getPathInfo();
-        if (pathInfo != null && pathInfo.equals(getExclusionPath())) {
-            chain.doFilter(req, resp);
-            return true;
+        if (isEmpty(pathInfo)) {
+            return false;
+        }
+        // Github will not follow redirects https://github.com/isaacs/github/issues/574
+        pathInfo = !pathInfo.endsWith("/") ? pathInfo + '/' : pathInfo;
```

Comment:
please swap arguments to exclude negative logic 
`pathInfo = pathInfo.endsWith("/") ? pathInfo : pathInfo + '/' ;`